presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -823,6 +851,29 @@ private void assertSTGeometryN(String wkt, int index, String expected)
         assertFunction("ST_ASText(ST_GeometryN(ST_GeometryFromText('" + wkt + "')," + index + "))", VARCHAR, expected);
     }
 
+    @Test
+    public void testSTInteriorRingN()
+    {
+        assertInvalidFunction("ST_AsText(ST_InteriorRingN(ST_GeometryFromText('POINT EMPTY'), 0))", "ST_InteriorRingN only applies to POLYGON. Input type is: POINT");
```

Comment:
let's make a couple of changes to improve readability;

* drop `ST_AsText`
* extract helper method `assertInvalidInteriorRingN`